users/discord/App.js

```diff
@@ -17,9 +17,46 @@ export const usersData = {
 const urlParams = new URLSearchParams(window.location.search);
 
 let activeTab = urlParams.get('tab') ?? 'in_discord';
-
+const INITIAL_USERS = 10;
+let isLoading = false;
+let currentPage = 1;
 let showUser = 0;
-usersData[activeTab] = await getUsers(activeTab);
+
+// usersData[activeTab] = await getUsers(activeTab);
```

Comment:
remove this if not needed

Reply:
Currently testing with mockdata and that function is still being used to fetch the actual data